torch/_dynamo/variables/torch.py

```diff
@@ -236,15 +236,21 @@ def call_function(
         unspec_python_args = check_unspec_python_args(args, kwargs)
         options = VariableTracker.propagate(self, args, kwargs.values())
 
+        def get_full_name(value):
+            if hasattr(value, "__module__") and hasattr(value, "__name__"):
+                if value.__module__ is not None and value.__name__ is not None:
+                    return value.__module__ + "." + value.__name__
```

Comment:
nit: you could do `f"{value.__module__ }.{value.__name__}"` instead?

Reply:
I think this change is too risky after all. Let's put it as a TODO in a standalone PR.